src/openclsim/core/movable.py

```diff
@@ -187,18 +187,77 @@ def v(self):
 
 class Routable(Movable, Locatable):
     """Mixin class: Something with a route (networkx node list format)
-    route: a list of node ids (available on env.graph) or geometries (shapely.Geometry)
+    route: a list of node ids (available on env.graph)
+    path: a linestring used to sail over
     """
```

Comment:
can we do this in the numpy docstring style, just like all the other docstring?

```
    """
    Mixin class: Something with a route (networkx node list format).

    Parameters
    ----------
    route
        A list of node ids (available on env.graph)
    path
        A linestring used to sail over
    """
```

Reply:
I agree to adheere to  numpy docstring style